etc/column/column.conf

```diff
@@ -0,0 +1,10 @@
+[DEFAULT]
+# Log file
+log_file = /var/log/column.log
+
+# Log info level
+log_level = DEBUG
+
+[API]
+server = 0.0.0.0
```

Comment:
I think this is fine as a default value. It's deployer's duty to figure out the actual interface service should be bound to, we can't make this decision for him/her. At the same time, we need something here for testing/development.